.github/workflows/build-verification.yml

```
@@ -29,326 +29,324 @@ jobs:
-    
+
     services:
       qdrant:
         image: qdrant/qdrant:latest
         ports:
           - 6333:6333
           - 6334:6334
-        # Remove health check for now to let the container start properly
-        # We'll check connectivity in a separate step
-    
+        healthcheck:
```

actionlint (1.7.4)

unexpected key "healthcheck" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)